src/hooks/useTranslation.tsx

```diff
@@ -15,8 +15,8 @@ const translationMap = {
 	'info.edit_tag_no_permissions':
 		"You don't have permission to edit the tags on the selected {{resource}}",
 	'info.no_selected': "You haven't selected anything yet",
-	'info.not_defined': 'Not defined',
-	'info.learn_more': 'Learn more about {{title}}',
+	'info.click_to_read_more': 'Click to read more about {{title}}',
```

Comment:
```suggestion
	'info.click_to_read_more': 'Read more about {{title}}',
```

See https://www.dont-click-here.com/